app/util/TraceIdExt.scala

```diff
@@ -34,7 +35,7 @@ object TraceIdExt {
     traceIdStringsFromQueryParameter()
       .flatMap { seq =>
         val set = seq.map(TraceId.apply).toSet
-        if (set.size > 1) Logger(this.getClass).error(s"Multiple traceIds in the URL. [${set.mkString(", ")}]")
+        if (set.size === 1) Logger(this.getClass).error(s"Multiple traceIds in the URL. [${set.mkString(", ")}]")
```

Comment:
this looks wrong

Reply:
whoops! I changed it cos of a wartremover warning and then didn't need it after all and changed it back to the wrong thing 😬 I'll change it back